{{cookiecutter.repostory_name}}/bin/notify.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import os
+from datetime import datetime
+
+import requests
+
+msg_template = "*{timestamp}*\n{author}\n{changelog}"
+
+
+class SlackNotifier:
+    url = "https://slack.com/api/chat.postMessage"
+
+    def __init__(self, token, channel):
+        if not channel or not token:
+            raise Exception("Missing token or channel")
+
+        self.token = token
+        self.channel = channel
+
+    def notify(self, text):
+        print(f"Sending message to channel {self.channel}")
+        r = requests.post(
+            self.url,
+            data={
+                "text": text,
+                "channel": self.channel,
+                "token": self.token,
+            },
+        )
+        r.raise_for_status()
```

Comment:
single letter variables can save you a variable name completion keystroke but make things harder for engineers who will try to fix something that doesn't work right in the middle of the night. It's `response`, lets call it that way.